vm/systemSmartContracts/delegation.go

```diff
@@ -1736,7 +1736,7 @@ func (d *delegation) unDelegate(args *vmcommon.ContractCallInput) vmcommon.Retur
 		return vmcommon.UserError
 	}
 
-	d.createAndAddLogEntry(args, valueToUnDelegate.Bytes(), remainedFund.Bytes(), globalFund.TotalActive.Bytes())
+	d.createAndAddLogEntry(args, valueToUnDelegate.Bytes(), remainedFund.Bytes(), big.NewInt(0).Bytes(), globalFund.TotalActive.Bytes())
```

Comment:
`big.NewInt(0).Bytes()` will return an empty byte slice. Is this correct?

Reply:
Yes.